### ISSUE TITLE
Can't run package json script 

### ISSUE THREAD

**Issue by mxxnseat** [clinicjs/node-clinic#462]
## Expected Behavior

I can run package.json script from command line with clinic heapprofiler
`clinic heapprofiler -- yarn test`

## Current Behavior

I can't run(or profile is empty) package.json script from command line with clinic heapprofiler `clinic heapprofiler -- yarn test`
I get error - **Clinic.js must be called with a `node` command line: `clinic heap-profiler -- node script.js`**

I also tried to do next but profile is empty
`clinic heapprofiler -- node /home/mxxnseat/.nvm/versions/node/v18.16.1/bin/yarn test`
![image](https://github.com/clinicjs/node-clinic/assets/69489064/11033353-a980-47de-84e2-5e26266854ba)

My package.json test script - `yarn clean && TS_NODE_PROJECT=tsconfig.spec.json nyc -t .nyc_output_integration mocha --config test/integration/.mocharc.yaml`

Also I can't use `node /node_modules/[module].js` as I use pnp node_linker in yarn config and I have zipped modules

## Environment

* Clinic.js version: 5.0.0

* Node.js version: v18.16.1
* Npm version: 9.5.1
* yarn version: 3.6.0
* typescript version: 4.6.4
* Mocha version: 10.2.0

* Operating system and version: Ubuntu 22.04

**Comment by alfonsograziano** [comment]
Hey! This sounds like a really specific use-case. Can you please provide a minimal repository to reproduce your issue? 

In every case, Clinic.js relies heavily on Node.js core instrumentation. That's why you need to run it with a `node` command.
If you run any other command, Clinic will not be able to read the data coming from the Node process

**Comment by tushar32** [comment]
I also have the same issue.

My application is running with below command
ts-node -r tsconfig-paths/register src/index.ts

But clinic js enforces to use -- node so I dont think that it supports ts-node..

@RafaelGSS  is it true?

**Comment by alfonsograziano** [comment]
Yes correct. What you can do is to compile your Typescript in a dist folder, and then run  `node dist/index.ts` with clinic.
You just need to perform two steps instead of one :)

**Comment by RafaelGSS** [comment]
@tushar32 you can run it by:

```console
$ node ./node_modules/.bin/ts-node -r tsconfig-paths/register src/index.ts
```

**Comment by tushar32** [comment]
Yes I run 
$  clinic doctor --autocannon [ -m GET 'http://localhost:5000/api/v1/data-elements/requests?status=PENDING&isRequester=false'] -- node build/index.js

Command but it does nothing no log or error.

When I run node builod/index.js it works fine

**Comment by tushar32** [comment]
This works but there is another senario 

I am facing error 

<img width="664" alt="Screenshot 2024-04-04 141203" src="https://github.com/clinicjs/node-clinic/assets/30208305/94ba308b-590e-4ff5-a493-ba4ed410e1b5">


After running clinic command it is not able to resolve @lib path in a file
